tests/e2e/sdk-to-pod.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+echo "Starting the script to copy the SDK to the pod"
```

Comment:
Is this script related to `tests/e2e/codeflare_sdk-0.0.0.dev0-py3-none-any.whl`?

Reply:
Hmm no, good catch I forgot to delete that wheel. In the installation of the codeflare-sdk for this test there is no wheel involved, so `poetry build` is never performed.